infra/app/web.bicep

```diff
@@ -0,0 +1,18 @@
+param environmentName string
+param location string = resourceGroup().location
+param appServicePlanId string
+
+param serviceName string = 'web'
+
+module web '../core/host/appservice-dotnet.bicep' = {
```

Comment:
Make sure you have all the latest /bicep/core from azure-dev. It was recently updated.